src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -714,6 +714,12 @@ protected DynamoModel(IStartConfiguration config)
                     PreferenceSettings.IsFirstRun = isFirstRun;
                 }
             }
+
+            if (PreferenceSettings.IsFirstRun && !IsTestMode)
```

Comment:
why not add them in test mode?

Reply:
Notice trust warng feature is not enabled in test mode